frontend/src/pages/Documentation.jsx

```diff
@@ -1,58 +1,17 @@
-import {
-  cyan,
-  grey,
-  pink,
-} from '@material-ui/core/colors'
-import {
-  createMuiTheme,
-  MuiThemeProvider,
-  StylesProvider,
-} from '@material-ui/core/styles'
 import React from 'react'
 import ReactDOM from 'react-dom'
-import 'Styles/documentation.scss'
-import DocumentationComponent from '../components/documentation'
-
-const {
-  500: cyan500,
-} = cyan
-const {
-  A200: pinkA200,
-} = pink
-const {
-  100: grey100,
-  300: grey300,
-  400: grey400,
-  500: grey500,
-  900: darkBlack,
-} = grey
-const white = `#ffffff`
-const fullBlack = `#000000`
-
-const muiTheme = createMuiTheme({
-  fontFamily: `Roboto, sans-serif`,
-  palette: {
-    primary1Color: `#434343`,
-    primary2Color: `#434343`,
-    primary3Color: grey400,
-    accent1Color: pinkA200,
-    accent2Color: grey100,
-    accent3Color: grey500,
-    textColor: darkBlack,
-    alternateTextColor: white,
-    canvasColor: white,
-    borderColor: grey300,
-    pickerHeaderColor: cyan500,
-    shadowColor: fullBlack,
-  },
-})
+import { NavBar } from "../components/layout/Items.jsx"
+import SwaggerUI from "swagger-ui-react"
+import "Styles/common/uclapi.scss"
+import "Styles/navbar.scss"
+import "Styles/documentation.scss"
+import "swagger-ui-react/swagger-ui.css"
 
 const Documentation = () => (
-  <StylesProvider injectFirst>
-    <MuiThemeProvider theme={muiTheme}>
-      <DocumentationComponent />
-    </MuiThemeProvider>
-  </StylesProvider>
+  <div className='vertical-padding'>
+    <NavBar isScroll={false} />
+    <SwaggerUI url="https://cdn.jsdelivr.net/gh/uclapi/uclapi-openapi/uclapi.json" />

```

Comment:
Can this be built statically? The reason I ask is, firstly for performance reasons, but more importantly is for consistency. At what point is `https://cdn.jsdelivr.net/gh/uclapi/uclapi-openapi/uclapi.json` updated? It seems like either the documentation should be generated as part of a build step or be part of the statics?

Reply:
Yeah I agree statically building is best -- I should have really marked this as WIP after our meeting sorry! 

I haven't had much time to look into the best way to build it statically and join it with the webpack build yet, but I'll try looking into it ASAP - openapi-generator has a [dynamic-html docs generation option](https://openapi-generator.tech/docs/generators/dynamic-html) which looks promising for us!

Reply:
It's part of the build process now!